package.json

```diff
@@ -21,6 +21,7 @@
     "e2e:cy:run:regression": "yarn --cwd ./projects/storefrontapp-e2e-cypress run cy:run:regression",
```

Comment:
Can we add:
 - build - https://github.com/SAP/cloud-commerce-spartacus-storefront/blob/develop/package.json#L9
 - do we need to add a script to start cds e2e test(s)? we should not pollute this file, and add only the scripts that are going to be executed from the local to run cds e2e test(s) _and_ that are going to be used by the CI to run cds e2e test(s). Maybe it's even the same command, not sure.